src/Core/Models/Business/OrganizationLicense.cs

```diff
@@ -54,11 +54,6 @@ public OrganizationLicense(Organization org, SubscriptionInfo subscriptionInfo,
         SmSeats = org.SmSeats;
         SmServiceAccounts = org.SmServiceAccounts;
 
-        // Deprecated. Left for backwards compatibility with old license versions.
-        LimitCollectionCreationDeletion = org.LimitCollectionCreationDeletion;
-        AllowAdminAccessToAllCollectionItems = org.AllowAdminAccessToAllCollectionItems;
-        //
-
```

Comment:
I think we still need this. Self-host users often lag several versions behind and old installs will still need this value to flow through from cloud via the license file. In practice we could just leave it here forever with a comment, and using the transitional `OR` logic.